packages/build-tools/src/utils/fingerprint.ts

```diff
@@ -0,0 +1,175 @@
+export type FingerprintSource = HashSource & {
+  /**
+   * Hash value of the `source`.
+   * If the source is excluding by `Options.dirExcludes`, the value will be null.
```

Comment:
Would be useful to document what hash algorithm is used.

Reply:
I'll add a comment to this file - the function is a direct copy/paste from the fingerprint repo where it also has tests.
https://github.com/expo/expo/pull/29709